src/test/java/com/zendesk/maxwell/MaxwellTestSupport.java

```diff
@@ -305,4 +324,19 @@ public static void requireMinimumVersion(MysqlIsolatedServer server, MysqlVersio
 		// skips this test if running an older MYSQL version
 		assumeTrue(server.getVersion().atLeast(minimum));
 	}
+
+	private static int getIncompleteBootstrapTaskCount(Maxwell maxwell, String clientID) throws SQLException {
+		try ( Connection cx = maxwell.context.getMaxwellConnection() ) {
```

Comment:
This `try` block caught an Exception before it was moved into the tests.  Was that required?

If no exceptions are going to be caught, I don't think there is a need for the `try` (my java is very rusty though).

Reply:
Still useful, it's a try-dispose: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Thanks.  That's a new java addition since my java days.